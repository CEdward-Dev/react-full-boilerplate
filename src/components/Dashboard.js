import React from 'react';

const Dashboard = () => (
    <div>
        Dashboard component content
    </div>
);

export default Dashboard;