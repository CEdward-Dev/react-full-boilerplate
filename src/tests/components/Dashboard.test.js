import { shallow } from 'enzyme';
import React from 'react';
import Dashboard from '../../components/Dashboard';

test('Should render the Dashboard page', () => {
    const wrapper = shallow(<Dashboard />);
    expect(wrapper).toMatchSnapshot();
});