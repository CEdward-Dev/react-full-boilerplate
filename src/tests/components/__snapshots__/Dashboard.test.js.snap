// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the Dashboard page 1`] = `
<div>
  Dashboard component content
</div>
`;
